Staking Rewards - Research Methodology Development - Milestone 1 (#2484)
w3f/Grants-Program: applications/staking_rewards.md

# Validator Performance - Research & Methodology Development

- **Team Name:** Staking Rewards
- **Payment Address:** Payment Details will be sent to grants@web3.foundation privately.
- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2

### Overview

Staking Rewards proposes a comprehensive research initiative to enhance Polkadot’s staking ecosystem by developing a robust and unbiased methodology to rank validators and improve staking accessibility. This proposal also includes deploying intuitive staking tools through Stake App and executing a marketing campaign targeting over 250,000 crypto-focused users. The initiative aims to simplify staking, boost participation, and increase the visibility of Polkadot’s staking features and liquid staking solutions.

### Project Details

### Milestone 1 Research & Methodology Development

**Objective:** Develop a comprehensive, data-driven framework to rank validators for the Polkadot network, which will form the foundation for selecting the Top 16 Decentralized Nodes (DNs) from the Polkadot Decentralized Nodes Program for the Default Nomination Pool in the Stake App.

> **Deliverables for Milestone 1:**
> 
> | Number  | Deliverable | Specification |
> | ------- | ---------- | ------------- |
> | 0.      | Article    | We will publish an **article**/workshop that explains [...] (what was done/achieved as part of the grant). The article will include: |
> |         |            | - **Metrics:** Performance & Reliability (uptime, block production efficiency, latency, slash history), Staking & Rewards (commission rate, APY, minimum stake), Reputation & Trust (identity verification, slash-free period, reputation), Decentralization & Inclusion (geographical distribution, operator diversity), Technical Setup (hardware specs, client version), Engagement & Transparency (governance participation, validator communication), Financial & Risk (stake distribution, oversaturation risk, fee stability). |
> |         |            | - **Data Sources:** Analysis of on-chain and off-chain data sources. |
> |         |            | - **Validator Scoring:** Preliminary scoring model for ranking validators based on weighted metrics. |
> |         |            | - **DN Selection:** A ranked list of the **Top 16 Decentralized Nodes (DNs)** for inclusion in the Default Nomination Pool. |
> |         |            | - **Research Report:** Results published in the form of a research report. |

Reviewer: ```suggestion

```

Moving the deliverables table back to the roadmap section

### Team Code Repos

GitHub: https://github.com/stakingrewards
Product: https://www.stakingrewards.com/

### Team LinkedIn Profiles (if available)

https://www.linkedin.com/company/stakingrewards/people/

## Development Status :open_book:

Currently Staking Rewards has developed a dedicated Asset Page featuring Polkadot Validators:
https://www.stakingrewards.com/asset/polkadot

## Development Roadmap :nut_and_bolt:

Milestone 1 is the only Research Development Milestone. For future milestones, it needs to be evaluated if further research development is needed. Currently this is not planned.

### Overview

- **Total Estimated Duration:** 2 months
- **Full-Time Equivalent (FTE):**  2 FTE
- **Total Costs:** 15.000 USD
- **DOT %:** 50 %

> 
> 

Reviewer: ```suggestion

**Deliverables for Milestone 1:**

| Number  | Deliverable | Specification |
| ------- | ---------- | ------------- |
| 0.      | Article    | We will publish an **article**/workshop that explains [...] (what was done/achieved as part of the grant). The article will include: |
| 1.      | Metrics | Performance & Reliability (uptime, block production efficiency, latency, slash history), Staking & Rewards (commission rate, APY, minimum stake), Reputation & Trust (identity verification, slash-free period, reputation), Decentralization & Inclusion (geographical distribution, operator diversity), Technical Setup (hardware specs, client version), Engagement & Transparency (governance participation, validator communication), Financial & Risk (stake distribution, oversaturation risk, fee stability). |
| 2.      | Data Sources | Analysis of on-chain and off-chain data sources. |
| 3.      | Validator Scoring | Preliminary scoring model for ranking validators based on weighted metrics. |
| 4.      | DN Selection | A ranked list of the **Top 16 Decentralized Nodes (DNs)** for inclusion in the Default Nomination Pool. |
| 5.      | Research Report | Results published in the form of a research report. |

```

### Team Code Repos

GitHub: https://github.com/stakingrewards
Product: https://www.stakingrewards.com/

### Team LinkedIn Profiles (if available)

https://www.linkedin.com/company/stakingrewards/people/

## Development Status :open_book:

Currently Staking Rewards has developed a dedicated Asset Page featuring Polkadot Validators:
https://www.stakingrewards.com/asset/polkadot

## Development Roadmap :nut_and_bolt:

Milestone 1 is the only Research Development Milestone. For future milestones, it needs to be evaluated if further research development is needed. Currently this is not planned.

### Overview

- **Total Estimated Duration:** 2 months
- **Full-Time Equivalent (FTE):**  2 FTE
- **Total Costs:** 15.000 USD
- **DOT %:** 50 %

**Deliverables for Milestone 1:**

| Number  | Deliverable | Specification |
| ------- | ---------- | ------------- |
| 0a.     | Copyright and Licenses | CC BY 4.0 |

> | 0e.     | Article    | We will publish an **article**/workshop that explains what was done/achieved as part of the grant. The article will include: |
> |         |            | - **Metrics:** Performance & Reliability (uptime, block production efficiency, latency, slash history), Staking & Rewards (commission rate, APY, minimum stake), Reputation & Trust (identity verification, slash-free period, reputation), Decentralization & Inclusion (geographical distribution, operator diversity), Technical Setup (hardware specs, client version), Engagement & Transparency (governance participation, validator communication), Financial & Risk (stake distribution, oversaturation risk, fee stability). |
> |         |            | - **Data Sources:** Analysis of on-chain and off-chain data sources. |
> |         |            | - **Validator Scoring:** Preliminary scoring model for ranking validators based on weighted metrics. |
> |         |            | - **DN Selection:** A ranked list of the **Top 16 Decentralized Nodes (DNs)** for inclusion in the Default Nomination Pool. |
> |         |            | - **Research Report:** Results published in the form of a research report. |

Reviewer: ```suggestion
| 1.     | Article    | We will publish an **article**/workshop that explains what was done/achieved as part of the grant. The article will include: |
| 1a.       |            | - **Metrics:** Performance & Reliability (uptime, block production efficiency, latency, slash history), Staking & Rewards (commission rate, APY, minimum stake), Reputation & Trust (identity verification, slash-free period, reputation), Decentralization & Inclusion (geographical distribution, operator diversity), Technical Setup (hardware specs, client version), Engagement & Transparency (governance participation, validator communication), Financial & Risk (stake distribution, oversaturation risk, fee stability). |
| 1b.       |            | - **Data Sources:** Analysis of on-chain and off-chain data sources. |
| 1c.        |            | - **Validator Scoring:** Preliminary scoring model for ranking validators based on weighted metrics. |
| 1d.        |            | - **DN Selection:** A ranked list of the **Top 16 Decentralized Nodes (DNs)** for inclusion in the Default Nomination Pool. |
| 2.        | Research Report | Results published in the form of a research report. |
```

Hey @Romao08, thanks for the changes. The table is a bit unclear, so I made a change suggestion. This basically states that the article will contain everything from `a.` to `d.` 

Since you again included an article _and_ a report, can you clarify what the difference between the two is?

Author: perfect! Basically the article was supposed to be a summary of the report, but we can remove the article and do the report only. 

we can change the first point from article to report and remove the last point

Reviewer: I would much prefer if we also had an easily digestible summary of the report, so I would really leave it in.

### Team Code Repos

GitHub: https://github.com/stakingrewards
Product: https://www.stakingrewards.com/

### Team LinkedIn Profiles (if available)

https://www.linkedin.com/company/stakingrewards/people/

## Development Status :open_book:

Currently Staking Rewards has developed a dedicated Asset Page featuring Polkadot Validators:
https://www.stakingrewards.com/asset/polkadot

## Development Roadmap :nut_and_bolt:

Milestone 1 is the only Research Development Milestone. For future milestones, it needs to be evaluated if further research development is needed. Currently this is not planned.

### Overview

- **Total Estimated Duration:** 2 months
- **Full-Time Equivalent (FTE):**  2 FTE

> - **Total Costs:** 15.000 Euro

Reviewer 2: @Romao08 Thanks for confirming that USDC is fine. Can you please update total costs here to indicate USD?
```suggestion
- **Total Costs:** 15.000 USD
```

### Milestone 1 Research & Methodology Development

- **Estimated duration:** 2 month
- **FTE:**  2

> - **Costs:** 15,000 Eur

Reviewer 2: ```suggestion
- **Costs:** 15,000 USD
```

https://www.stakingrewards.com/asset/polkadot

## Development Roadmap :nut_and_bolt:

Milestone 1 is the only Research Development Milestone. For future milestones, it needs to be evaluated if further research development is needed. Currently this is not planned.

### Overview

- **Total Estimated Duration:** 2 months
- **Full-Time Equivalent (FTE):**  2 FTE
- **Total Costs:** 15.000 USD
- **DOT %:** 50 %

### Milestone 1 Research & Methodology Development

- **Estimated duration:** 2 month
- **FTE:**  2
- **Costs:** 15,000 USD

**Objective:** Develop a comprehensive, data-driven framework to rank validators for the Polkadot network, which will form the foundation for selecting the Top 16 Decentralized Nodes (DNs) from the Polkadot Decentralized Nodes Program for the Default Nomination Pool in the Stake App.

**Deliverables for Milestone 1:**
| Number  | Deliverable       | Specification                                                                                                |
| ------- | ----------------- | ------------------------------------------------------------------------------------------------------------ |

> | 0.      | Article           | We will publish an **article**/workshop that explains [...] (what was done/achieved as part of the grant). (Content, language, and medium should reflect your target audience described above.)  |

Reviewer: Please update the specification to match your goals.

| 1.      | Metrics           | List of metrics, categorized into: **Performance and Reliability:** Uptime, block production efficiency, latency, slash history. **Staking and Rewards:** Commission rate, APY, minimum stake requirement. **Reputation and Trust:** Identity verification, slash-free period, reputation. **Decentralization and Inclusion:** Geographical distribution, operator diversity. **Technical Setup:** Hardware specifications, client version. **Engagement and Transparency:** Governance participation, validator communication. **Financial and Risk:** Stake distribution, oversaturation risk, fee stability.                                                                                  |
| 2.      | Data Sources      | Analysis of on-chain and off-chain data sources.                                                             |
| 3.      | Validator Scoring | Preliminary scoring model for ranking validators based on weighted metrics.                                  |
| 4.      | DN Selection      | A ranked list of the **Top 16 Decentralized Nodes (DNs)** for inclusion in the Default Nomination Pool.      |

> | 5.      | Research Report   | Results will be published in the form of a **Research Report**.                                              |

Reviewer: Is this different from deliverable 0? If not, please remove either one.
